suse_openstack_cloud

```diff
@@ -396,13 +396,12 @@ fi
 section_header "RabbitMQ broker"
 
 if [ -e /usr/sbin/rabbitmqctl ]; then
```

Comment:
I would like leave `/usr/sbin/rabbitmqctl` it is shorter and nice to read. Both files are installed by rabbitmq-server packade